doc/DeveloperGuide.md

```diff
@@ -88,6 +88,45 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Add person
+
+**MSS**
+
+1. User requests to add person
+2. AddressBook adds the person <br>
+Use case ends.
+
+**Extensions**
+
+1a. The input is invalid
+
+> Use case ends
+
+
+#### Use case: Sort
+
+**MSS**
+
+1. User requests to sort AddressBook by name of persons
+2. AddressBook sorts itself
+3. AddressBook shows a sorted list of persons <br>
```

Comment:
The interaction between user and system should ideally be a to and fro interaction.
System should not be written in twice in a row.
Example: 
1. User does something on system
2. System reacts to user's input
3. User  "blah blah"
4. System  "blah blah"